DESCRIPTION

```diff
@@ -0,0 +1,15 @@
+Package: testpackage
+Title: C samples for ReGenerator
+Version: ##VERSION##
+Date: ##DATE##
+Authors@R: c(person("Magdalena", "Pastula", role = "aut",
+					email = "pastula@student.agh.edu.pl"),
+             person("Pawel", "Tyszko", role = "aut",
+					email = "tyszko@student.agh.edu.pl"),
+             person("Leszek", "Grzanka", role = c("aut", "cre", "rev", "ths"),
+					email = "grzanka@agh.edu.pl"))
+Description: A bunch of sample code written to test the capabilities of ReGenerator
+Depends: R (>= 3.1.0)
+License: MIT
```

Comment:
There is no license yet for ReGenerator, for consistency you should add there MIT as well.